.github/workflows/release-ts-api-package.yml

```diff
@@ -1,16 +1,55 @@
-name: Release Ts API package
+# Reference: ci.yml
+name: Release Ts API Package
+
 on:
   workflow_dispatch:
-
+      inputs:
+        tag:
+          description: 'Publish tag'
```

Comment:
Maybe a better desc would be `release docker image tag`

Reply:
Modified!